enterprise/frontend/src/embedding-sdk/lib/interactive-question/run-question-on-load.ts

```diff
@@ -35,8 +40,26 @@ export const runQuestionOnLoadSdk =
     const originalQuestion =
       originalCard && new Question(originalCard, metadata);
 
+    let question = new Question(card, metadata);
+
+    question = syncCardParametersWithTemplateTags(question);
```

Comment:
This gets the template tag parameters from the card, then apply it with `setParameters` internally, so the question object is aware of what parameters are there. This allows us to use `setParameterValues` to assign the SQL parameter values later.